shared/src/proxies/trialSessions/getTrialSessionsForJudgeProxy.js

```diff
@@ -9,6 +9,6 @@ const { get } = require('../requests');
 exports.getTrialSessionsForJudgeInteractor = (applicationContext, judgeId) => {
   return get({
     applicationContext,
-    endpoint: `/judges/${judgeId}/trial-sessions`,
+    endpoint: `/judges/${judgeId}/trial-sessions?fields=trialLocation,trialSessionId,sessionStatus,startDate`,
```

Comment:
I love the idea of using select fields in our endpoints, are these being used right now though?

Reply:
not sure what you mean?  are these properties being used?  the ability to pass `fields` in the query string has existed in our code base for 3 years, but I don't think it's been used at all

Reply:
My question was, how are the fields being consumed/used by the backend? Right now it appears to me like they aren't

Reply:
this logic which looks for fields is in the apiGatewayHelper, so you can pass this query string to any endpoint to filter down the data set.

Reply:
Got it, thank you!